src/app/public/modules/page-summary/page-summary-key-info.component.ts

```diff
@@ -1,5 +1,9 @@
 import { Component } from '@angular/core';
 
+/**
+ * Specifies important information in the key information section of the
+ * page summary.
```

Comment:
@Blackbaud-ToddRoberts, are you good with dropping the qualifying info: `This section can display any type of content, but it generally highlights a key information block such as important summary numbers.`? Or should we port this over to the Design tab?